client/src/auth/saga.js

```diff
@@ -15,11 +17,11 @@ export function* handleLogin({ payload: { history } }) {
     const res = yield axios.get(`/player/${currentUser.id}`);
     const { id, name } = res.data.player;
     yield put(authActions.loginSuccess({ id, name }));
-    yield put(alertActions.addAlert('Login successful!', 'success'));
+    yield put(homeActions.refreshActiveGames({ id: currentUser.id }));
```

Comment:
This should be a `useEffect` [hook](https://reactjs.org/docs/hooks-reference.html#useeffect) in `Home`:

```jsx
const Home = props => {
  // other stuff...
  const dispatch = useDispatch();
  // because we pass nothing as an effect dependency (the second arg), 
  // this will run once when we first render Home
  useEffect(() => {
    dispatch(homeActions.refreshActiveGames({id: currentUser.id}));
  }, []);
}
```